### ISSUE TITLE
Non-Latin strings are not shown properly

### ISSUE THREAD

**Issue by salarmehr** [Codeception/Codeception#2892]
Running the following cept 

```
$I = new AcceptanceTester($scenario);
$I->amOnPage('/');
$I->see('ورود');

```

The output will be:

```
Acceptance Tests (1) ---------------------------------------------------------------------------------
Ensure that frontpage works (WelcomeCept)
Scenario:
* I am on page "/"
* I see "┘ê╪▒┘ê╪»"
 PASSED
------------------------------------------------------------------------------------------------------
```

I use ConEmu as my terminal app on windows 10.

**Comment by Naktibalda** [comment]
This could be fixed by https://github.com/Codeception/Codeception/pull/2831
Please install 2.1 branch with Composer and try again:
`"codeception/codeception": "2.1.x-dev",`

**Comment by DavertMik** [comment]
Or wait for 2.1.7 release which is coming in few next days

**Comment by salarmehr** [comment]
@DavertMik OK. I prefer a stable version. Thanks.

**Comment by DavertMik** [comment]
It is here now. Reopen if that didn't fix an issue

**Comment by salarmehr** [comment]
It is not solved yet in version 2.1.7
Please try the provided sample test. You should see 'ورود' instead of '"┘ê╪▒┘ê╪»"'

Thanks.
